src/app/Media/Codecs/AudioEncoder.cs

```diff
@@ -80,7 +78,7 @@ public AudioEncoder(bool includeLinearFormats = false, bool includeOpus = false)
 
             if(includeOpus)
             {
-                _supportedFormats.Add(new AudioFormat(111, AudioCodecsEnum.OPUS.ToString(), OPUS_SAMPLE_RATE, OPUS_CHANNELS, "useinbandfec=1"));
+                _supportedFormats.Insert(0, new AudioFormat(111, AudioCodecsEnum.OPUS.ToString(), OPUS_SAMPLE_RATE, OPUS_CHANNELS, "useinbandfec=1"));
             }

```

Comment:
I'm not so sure this is a good idea. Always making OPUS the preferred codec may not be desired. Maybe a better approach would be to allow callers to specify their desired codec order.

Reply:
i added opus support but it's never selected because PCMU is listed first. PCMU uses 160 bytes per frame for silence whereas opus uses 3. 
I think if people are calling the overload to specify opus is supported then it should be first not last in the list. Or provide an override where you can specify the list of codecs explicitly.